pkg/generator/ingress_translator.go

```diff
@@ -281,3 +287,27 @@ func mergeMapString(a, b map[string]string) map[string]string {
 	}
 	return merged
 }
+
+// insertProbe adds a AppendHeader rule so that any request going through a Gateway is tagged with
+// the version of the Ingress currently deployed on the Gateway.
+func insertProbe(ing *v1alpha1.Ingress) (string, error) {
```

Comment:
It looks like this function is exposed in the new version of knative.dev/pkg. I think we should be able to reuse it if we merge https://github.com/knative/net-kourier/pull/19

Reply:
opened 20 minutes ago! I will check if in that version is available.

Reply:
Some folks deleted it thinking that it was unused, and we reverted that change.

The bot kicks off on a cron, so should be same time every day.

Reply:
@mattmoor I see that it also makes it not backward compatible, and I don't know if breaking the compatibility for such a critical part (the readiness probe) makes sense... :(